src/stactools/naip/stac.py

```diff
@@ -121,30 +124,51 @@ def create_item(
         )
 
     if fgdc_metadata_href is not None:
-        fgdc_metadata_text = read_text(fgdc_metadata_href)
-        fgdc = parse_fgdc_metadata(fgdc_metadata_text)
-    else:
-        fgdc = {}
+        if year == "2020":
+            file_xpath = """gmd:identificationInfo/gmd:MD_DataIdentification/gmd:citation/\n
```

Comment:
this there supposed to be a `\n` iln this xpath?

Reply:
Removed